pgxn/neon/walproposer_pg.c

```diff
@@ -203,6 +205,16 @@ nwp_register_gucs(void)
 												 * GUC_LIST_QUOTE */
 							   NULL, assign_neon_safekeepers, NULL);
 
+	DefineCustomStringVariable(
+							   "neon.safekeeper_extra_conninfo_options",
+							   "libpq keyword parameters and values to apply to safekeeper connections",
+							   NULL,
+							   &safekeeper_extra_conninfo_options,
```

Comment:
nit: I would call it `safekeeper_conninfo_options` just because it's shorter, but both variants are fine to me. So can keep it as it is if you like it more

Reply:
Sure, why not.